definitions/checks/check_ipv6_disable.rb

```diff
@@ -0,0 +1,23 @@
+class Checks::CheckIpv6Disable < ForemanMaintain::Check
+  metadata do
+    label :check_ipv6_disable
+    description 'Check if ipv6.disable=1 is set at kernel level'
+  end
+
+  def run
+    grub_file = File.read('/etc/default/grub')
+
+    assert(!grub_file.include?("ipv6.disable=1"), error_message)
```

Comment:
I'd rather check `/proc/cmdline` as that shows what's currently booted vs "what could be booted if the user didn't fiddle with things"